src/etc/inc/interfaces.inc

```diff
@@ -3237,7 +3237,11 @@ alias {
 
 EOD;
     }
-
+    if (!isset($wancfg['dhcpoveridemtu'])) {
+        $dhclientconf .= <<<EOD
+supersede interface-mtu 0;      
+EOD;
```

Comment:
you don't need the heredoc syntax here - just use a double quoted string with \n

Reply:
it's ok either way. no need for further changes as I'll merge this manually